src/v/raft/consensus.h

```diff
@@ -845,7 +850,7 @@ class consensus {
     /// used for notifying when commits happened to log
     event_manager _event_manager;
     std::unique_ptr<probe> _probe;
-    ctx_log _ctxlog;
+    mutable ctx_log _ctxlog;
```

Comment:
is it an artifact from using the logger in a const method?

Reply:
yes, this is unfortunate

Reply:
I don't see it being used anywhere though?

Reply:
i dropped this change as it is not used